llvm/lib/CodeGen/GlobalISel/Combiner.cpp

```diff
@@ -27,6 +28,12 @@
 
 using namespace llvm;
 
+STATISTIC(NumOneIteration, "Number of functions with one iteration");
+STATISTIC(NumTwoIterations, "Number of functions with two iterations");
+STATISTIC(NumThreeIterations, "Number of functions with three iterations");
+STATISTIC(NumFourOrMoreIterations,
+          "Number of functions with four or more iterations");
```

Comment:
Not sure all this variety is needed? We can probably prune this from instcombine now

Reply:
One, Two and Three are hit in CTMark. I have removed Four.